hw1/filter.js

```diff
@@ -0,0 +1,11 @@
+function filter(...params) {
+    const arr = params;
+    const modifiedArr = [];
+
+    arr.filter((item, index) => {
```

Comment:
`arr.filter` already creates a new array of filtered values, so you could do something like:
```suggestion
const modifiedArr = arr.filter((item, index) => arr.indexOf(item) === index)